src/extension/webView.ts

```diff
@@ -86,10 +86,10 @@ async function getNpmVersion(): Promise<string> {
 
 async function isLegacyNpmVersion(): Promise<boolean> {
   const npmVersion = await getNpmVersion();
-  const matches = npmVersion.split('.')[0].match(/\d+/g);
+  const matches = npmVersion.match(/\d+\.\d+\.\d+/g);
```

Comment:
Why?

I think you confuse two things. Let's talk first I have a feeling you go in circles and do not really read / understand what I write.